src/components/Form/Stepper/Stepper.scss

```diff
@@ -14,6 +14,7 @@
     font-size: $size-l;
     font-weight: normal;
     color: $greyscale-20;
+    z-index: 10;
```

Comment:
You should not use hardcoded z-index. We have a function in _helpers that does that for you. If you need help let me know :)

Reply:
I didnt touch z-index :o